contracts/DCAPermissionsManager/DCAPermissionsManager.sol

```diff
@@ -37,6 +38,8 @@ contract DCAPermissionsManager is ERC721, EIP712, Governable, IDCAPermissionMana
   mapping(address => uint256) public nonces;
   mapping(uint256 => uint256) public lastOwnershipChange;
   mapping(uint256 => mapping(address => TokenPermission)) public tokenPermissions;
+  // Gets initialized as 1 so first deposit doesn't get gas hit
+  uint256 internal _burnCounter = 1;
```

Comment:
Is this too much of an opt? Maybe I'm "over engineering" a bit here :/ - wdyt @nchamo?

Reply:
Yeah, I think so. I would prefer for the first person to burn to pay the cost once, and make the code clearer